OpenTabletDriver.Plugin/Output/RelativeOutputMode.cs

```diff
@@ -59,7 +64,22 @@ public float Rotation
         /// <summary>
         /// The delay in which to reset the last known position in relative positioning.
         /// </summary>
-        public TimeSpan ResetTime { set; get; }
+        public TimeSpan ResetTime
+        {
+            set
+            {
+                if (value == TimeSpan.Zero)
+                {
+                    Log.Write("RelativeOutputMode", "Reset time cannot be 0", LogLevel.Error);
```

Comment:
Probably a good idea to send a system notification to the user
```suggestion
                    Log.WriteNotify("RelativeOutputMode", "Reset time cannot be 0", LogLevel.Error);
```